src/nodeProfileCfg.m

```diff
@@ -143,6 +145,7 @@
    
 end
 
+nodeMobility = ~all(timeSampleNode(:) == 1);
```

Comment:
obscure piece of code, either comment it or make it more easily readable. i think you are saying that nodeMobility is false (or 0?) if all entries of timeSampleNode==1 (note that timeSampleNode is  an Nx2 matrix, not a simple vector), and true (or 1?) if at least one entry of timeSampleNode is ~=1. is that it? maybe a more direct name could also help understand its meaning

Reply:
added comment